static/js/components/FollowupRequestSelectionForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import Form from "@rjsf/material-ui";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { fetchFollowupRequests } from "../ducks/followup_requests";
+
+const FollowupRequestSelectionForm = () => {
+  const dispatch = useDispatch();
+
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const handleSubmit = async ({ formData }) => {
+    setIsSubmitting(true);
+    await dispatch(fetchFollowupRequests(formData));
+    setIsSubmitting(false);
+  };
+
+  function validate(formData, errors) {
+    if (formData.start_date > formData.end_date) {
+      errors.start_date.addError(
+        "Start date must be before end date, please fix."
+      );
+    }
```

Comment:
Are there no other restrictions on time (past or future times etc)?

Reply:
Not really I think, the requests span windows of time.